### ISSUE TITLE
Encoding problem in RStudio viewer pane but not in rmarkdown html output

### ISSUE THREAD

**Issue by renkun-ken** [renkun-ken/formattable#30]
When a data frame with unicode characters (e.g. Chinese), the `formattable` is not correctly rendered in RStudio viewer pane but rendered correctly in html document produced by rmarkdown renderer.

```r
df <- data.frame(id = 1:2, name = c("小明", "小红"), age=c(10,20))
formattable(df)
```

The locale of the R session must be set as Chinese first using 

```r
Sys.setlocale(locale = "Chinese")
```

**Comment by renkun-ken** [comment]
I trace the execution in `formattable:::as.htmlwidget.formattable` and the encoding is well preserved everywhere before `htmlwidgets::createWidget` is finally called. Then I guess the problem occurs in this function.

**Comment by renkun-ken** [comment]
The encoding problem is fixed in v0.2 branch.
